gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -72,6 +72,7 @@ end
 
 local function PreqLabels(parent, x, y)
 	local tbl = {}
+	local ply = LocalPlayer()
```

Comment:
maybe rename `ply` to `client` to keep it consitent with our code and make it clear which player is meant.